infrastructure/lambda/template.yaml

```diff
@@ -271,6 +274,14 @@ Resources:
           KeyStoreSecret: experian/keystore
           KeyStorePassword: experian/keystore-password
           WaspURLSecret: experian/iiq-wasp-service
+          WaspURLParameterName:
```

Comment:
I'm not following why we need to do this rather than accessing `experian/iiq-wasp-service/{clientId}` ?

Reply:
Doing the !Ref is passing the name i.e. `experian/iiq-wasp-service/{clientId}` and then in the code we call SSM. 

I have updated it so that all the environments use the Ref!